README.md

```diff
@@ -58,30 +57,26 @@ curl -s "https://raw.githubusercontent.com/pystardust/ani-cli/master/hist_transi
 ```
 It doesn't work for all anime, but the ones it can't find will print out alongside their episode numbers. In the end clean up: `rm -rf ./ani-cli`
 
-## Install
+## Install 
 
-### Arch
-
-Also consider ani-cli-git
-
-```sh
-yay -S ani-cli
-```
-
-### Other native packages
-
-[![Packaging status](https://repology.org/badge/vertical-allrepos/ani-cli.svg)](https://repology.org/project/ani-cli/versions)
+How installation works:
+- removal of previous installation
+- downloading the essestial files to the appropriate share/ani-cli/ directory for your OS
+- creating a symbolic link from aforementioned share/ani-cli/ani-cli binary to the appropriate bin/ani-cli
+- watch anime 💪🏻
 
 ### Linux
 
 Install dependencies [(See below)](#Dependencies)
 
 ```sh
-sudo curl -sL github.com/pystardust/ani-cli/raw/master/ani-cli -o /usr/local/bin/ani-cli &&
-sudo chmod +x /usr/local/bin/ani-cli
+sudo rm -rf "/usr/local/share/ani-cli" "/usr/local/bin/ani-cli" && \
+sudo git clone -b "master" "https://github.com/pystardust/ani-cli.git" "/usr/local/share/ani-cli" && \
+sudo ln -sf "/usr/local/share/ani-cli/ani-cli" "/usr/local/bin/ani-cli"
 ```
+*Note that while while running the above command, you'll be asked for your password.*
```

Comment:
Not necessary. I assume people who find this project and run linux are aware of how sudo works